ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -572,17 +575,23 @@ private NamedBlobRecord run_get_v2(String accountName, String containerName, Str
               version);
         }
       }
+    } catch (Exception e) {
+      logger.error("MySql query {} received exception", query, e);
+      throw e;
```

Comment:
nit: we are having the same ```MySql query {} received exception``` error log in multiple places. It's better to put the method name or some unique thing in the string to differentiate between them